docs/source/release/version0.12.rst

```diff
@@ -0,0 +1,914 @@
+- :pr:`6920`: MAINT: Remove FutureWarnings
+- :pr:`6922`: ENH: Add tool to simpligy documenting API in release notes
+
+API Changes
+===========
+
+
+New Classes
+-----------
+* :class:`statsmodels.base._constraints.LinearConstraints`
+* :class:`statsmodels.stats.base.HolderTuple`
+* :class:`statsmodels.stats.meta_analysis.CombineResults`
+* :class:`statsmodels.stats.robust_compare.TrimmedMean`
+* :class:`statsmodels.tsa.deterministic.CalendarDeterminsticTerm`
+* :class:`statsmodels.tsa.deterministic.CalendarFourier`
+* :class:`statsmodels.tsa.deterministic.CalendarSeasonality`
+* :class:`statsmodels.tsa.deterministic.CalendarTimeTrend`
+* :class:`statsmodels.tsa.deterministic.DeterministicProcess`
+* :class:`statsmodels.tsa.deterministic.DeterministicTerm`
+* :class:`statsmodels.tsa.deterministic.Fourier`
+* :class:`statsmodels.tsa.deterministic.FourierDeterministic`
+* :class:`statsmodels.tsa.deterministic.Seasonality`
+* :class:`statsmodels.tsa.deterministic.TimeTrend`
+* :class:`statsmodels.tsa.deterministic.TimeTrendDeterministicTerm`
+* :class:`statsmodels.tsa.exponential_smoothing.base.StateSpaceMLEModel`
+* :class:`statsmodels.tsa.exponential_smoothing.base.StateSpaceMLEResults`
+* :class:`statsmodels.tsa.exponential_smoothing.ets.ETSModel`
+* :class:`statsmodels.tsa.exponential_smoothing.ets.ETSResults`
+* :class:`statsmodels.tsa.exponential_smoothing.ets.ETSResultsWrapper`
+* :class:`statsmodels.tsa.forecasting.theta.ThetaModel`
+* :class:`statsmodels.tsa.forecasting.theta.ThetaModelResults`
+* :class:`statsmodels.tsa.holtwinters._exponential_smoothers.HoltWintersArgs`
+* :class:`statsmodels.tsa.holtwinters._smoothers.HoltWintersArgs`
+* :class:`statsmodels.tsa.holtwinters.model.ExponentialSmoothing`
+* :class:`statsmodels.tsa.holtwinters.model.Holt`
+* :class:`statsmodels.tsa.holtwinters.model.SimpleExpSmoothing`
+* :class:`statsmodels.tsa.holtwinters.results.HoltWintersResults`
+* :class:`statsmodels.tsa.holtwinters.results.HoltWintersResultsWrapper`
+* :class:`statsmodels.tsa.statespace._cfa_simulation_smoother.cCFASimulationSmoother`
+* :class:`statsmodels.tsa.statespace._cfa_simulation_smoother.dCFASimulationSmoother`
+* :class:`statsmodels.tsa.statespace._cfa_simulation_smoother.sCFASimulationSmoother`
+* :class:`statsmodels.tsa.statespace._cfa_simulation_smoother.zCFASimulationSmoother`
+* :class:`statsmodels.tsa.statespace.cfa_simulation_smoother.CFASimulationSmoother`
+* :class:`statsmodels.tsa.statespace.news.NewsResults`
+* :class:`statsmodels.tsa.statespace.news.NewsResultsWrapper`
+
+
+
+Removed Classes
+---------------
+* ``statsmodels.tsa.holtwinters.ExponentialSmoothing``
+* ``statsmodels.tsa.holtwinters.Holt``
+* ``statsmodels.tsa.holtwinters.HoltWintersResults``
+* ``statsmodels.tsa.holtwinters.HoltWintersResultsWrapper``
+* ``statsmodels.tsa.holtwinters.SimpleExpSmoothing``
+* ``statsmodels.tsa.stattools.ResultsStore``
+
+
+
+New Methods
```

Comment:
this section looks wrong
The GEE methods are inherited from GLM and not used.
GEE is like GMM and doesn't have a likelihood function (except as a special cases)

afaik there was no related change to GEE
@kshedden

Reply:
These are just automatically generated placeholders and will be edited a lot before release.  That the wrong methods show up for some model is probably an indication that the inheritance is wrong.